.travis.yml

```diff
@@ -1,34 +1,45 @@
+sudo: false
 language: cpp
 compiler:
   - clang
   - gcc
 env:
-  - TARGET=debug
-  - TARGET=debug.nounity
-  # We can specify any combination of builds here, for example, to
-  # include release builds, too, uncomment the following lines.
-  #- TARGET=release
-  #- TARGET=release.nounity
+  global:
+    - BOOST_ROOT=$HOME/boost_1_59_0
+  matrix:
+    - TARGET=debug
+    - TARGET=debug.nounity
+    # We can specify any combination of builds here, for example, to
+    # include release builds, too, uncomment the following lines.
+    #- TARGET=release
+    #- TARGET=release.nounity
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    packages:
+    - python-software-properties
+    # We need gcc >= 4.8 for some c++11 features
+    - g++-4.8
```

Comment:
The symbol `4.8` is used three times.  You might consider extracting it as another variable `GCC_VERSION` so when we upgrade to 4.9 or whatever we only have to change one place...